yarl/_helpers_c.pyi

```diff
@@ -1,6 +0,0 @@
-from typing import Any
```

Comment:
No longer needed since `_helpers.py` will always use `_helpers_py` for typing which it better typed with a Generic